### PR TITLE
[thread-cert] fix case 5.3.10 intermittent failure

### LINKED ISSUE #6018: [github-actions] Cert_5_3_10_AddressQuery fails intermittently
**Describe the bug**

tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py fails intermittently.

**Console/log output**

- [logs_47592.zip](https://github.com/openthread/openthread/files/5748495/logs_47592.zip)
- [packet-verification-pcaps.zip](https://github.com/openthread/openthread/files/5748498/packet-verification-pcaps.zip)

### PR DESCRIPTION
**Pull request by FiveDimensions**
Resolves #6018

### CODE REVIEW

**Comment by FiveDimensions** [comment]
fix issue #6018

File: tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def test(self):
 
         # Send an ICMPv6 Echo Request from MED1 to ROUTER1 GUA PREFIX_1 address
         self.assertFalse(self.nodes[MED1].ping(router1_addr))
-        self.simulator.go(1)
+        self.simulator.go(3)
```
Comment (jwhui) [inline tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py:175]: The problem you identified in https://github.com/openthread/openthread/issues/6018#issuecomment-752850508 seems to indicate we need to add more time to the 580s. Is this still true?

Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py]: The failed packets show there is Router_1 ID in the leader's advertisements after Router_1 power off 580,  add routerselectionjitter=1 for Leader

Comment (jwhui) [reply tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py]: Why does setting the leader's `router_selection_jitter` to 1 resolve this issue?

**Comment by jwhui** [comment]
Closing stale PR.

**Pull request closed without merging**